src/gui_common/TweakedColourPicker.cs

```diff
@@ -35,66 +34,20 @@ public partial class TweakedColourPicker : ColorPicker
     private HSlider sliderROrH = null!;
     private HSlider sliderGOrS = null!;
     private HSlider sliderBOrV = null!;
+
+    // Alpha has all 3 elements to fully hide it
     private HSlider sliderA = null!;
+    private Control labelA = null!;
+    private Control spinboxA = null!;
     private Button pickerButton = null!;
-    private CustomCheckBox? hsvCheckBox;
-    private CustomCheckBox rawCheckBox = null!;
-    private Label htmlColourStart = null!;
-    private LineEdit htmlColourEdit = null!;
-    private HSeparator separator = null!;
-    private GridContainer? presetsContainer;
-    private TextureButton? addPresetButton;
 #pragma warning restore CA2213
 
-    private bool hsvButtonEnabled = true;
-    private bool rawButtonEnabled = true;
-    private bool presetsEditable = true;
-    private bool presetsVisible = true;
-
     // TODO:
     private ColorModeType mode = ColorModeType.Rgb;
```

Comment:
It looks like to me that this variable should be removed. This is not updated anywhere so the tooltips which depend on this end up incorrect when the user switches to HSV mode for example.